lib/rucio/daemons/badreplicas/minos.py

```diff
@@ -69,24 +68,27 @@ def minos(bulk=1000, once=False, sleep_time=60):
     hb_thread = threading.current_thread()
     heartbeat.sanity_check(executable=executable, hostname=hostname)
     heart_beat = heartbeat.live(executable, hostname, pid, hb_thread)
-    prepend_str = 'Thread [%i/%i] : ' % (heart_beat['assign_thread'], heart_beat['nr_threads'])
-    logging.info(prepend_str + 'Minos starting')
+    prefix = 'Thread [%i/%i] : ' % (heart_beat['assign_thread'], heart_beat['nr_threads'])
```

Comment:
Please consider splitting the logging changes into a separate commit and referencing issue #4220. Additionally, the message format should be `daemon_name[heartbeat['assign_thread']/heartbeat['nr_threads']]: …`

Reply:
I would prefer not to revert the changes already done, but point taken for next time. I'll fix the format.